Detectors/TRD/qc/include/TRDQC/Tracking.h

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file Tracking.h
+/// \brief Check the performance of the TRD in global tracking
+/// \author Ole Schmidt
+
+#ifndef O2_TRD_TRACKINGQC_H
+#define O2_TRD_TRACKINGQC_H
+
+#include "DataFormatsTRD/TrackTRD.h"
+#include "DataFormatsTRD/Tracklet64.h"
+#include "DataFormatsTRD/CalibratedTracklet.h"
+#include "DataFormatsTRD/Constants.h"
+#include "ReconstructionDataFormats/TrackTPCITS.h"
+#include "DataFormatsTPC/TrackTPC.h"
+#include "DetectorsBase/Propagator.h"
+#include "TRDBase/RecoParam.h"
+#include "TH1.h"
+
+#include "Rtypes.h"
+
+#include <gsl/span>
+
+namespace o2
+{
+
+namespace globaltracking
+{
+class RecoContainer;
+}
+
+namespace trd
+{
+
+struct TrackQC {
+  int type; // 0 TPC-TRD track; 1 ITS-TPC-TRD track
+  int nTracklets;
+  float chi2;
+  float reducedChi2;
+  float pt;
+  std::array<float, constants::NLAYER> findable{};
+  std::array<float, constants::NLAYER> trackX{};
+  std::array<float, constants::NLAYER> trackY{};
+  std::array<float, constants::NLAYER> trackZ{};
+  std::array<float, constants::NLAYER> trackSnp{};
```

Comment:
Can you add a brief comment what Snp is? And maybe what coordinate system is used for the helix: global / tracking=sector / chamber?